test/pipelinefinally_test.go

```diff
@@ -201,13 +247,16 @@ func TestPipelineLevelFinally_OneDAGTaskFailed_InvalidTaskResult_Failure(t *test
 		t.Fatalf("Error listing TaskRuns for PipelineRun %s: %s", pipelineRun.Name, err)
 	}
 
-	// expecting taskRuns for dagtask1, dagtask2, dagtask3 (with condition failure), dagtask5, finaltask1, finaltask2, and finaltaskconsumingdagtask5
-	if len(taskrunList.Items) != 7 {
+	// expecting taskRuns for dagtask1, dagtask2, dagtask3 (with condition failure), dagtask5, finaltask1, finaltask2,
+	// finaltaskconsumingdagtask5, finaltaskconsumingdagtask5inwhenexpressionstrue, finaltaskconsumingdagtask5executionstatusinwhenexpressionstrue
+	expectedTaskRunsCount := 9
+	if len(taskrunList.Items) != expectedTaskRunsCount {
 		var s []string
 		for _, n := range taskrunList.Items {
 			s = append(s, n.Labels["tekton.dev/pipelineTask"])
 		}
-		t.Fatalf("Error retrieving TaskRuns for PipelineRun %s. Expected 5 taskRuns and found taskRuns for: %s", pipelineRun.Name, strings.Join(s, ", "))
+		t.Fatalf("Error retrieving TaskRuns for PipelineRun %s. Expected %s taskRuns and found %s taskRuns for: %s",
+			pipelineRun.Name, strconv.Itoa(expectedTaskRunsCount), strconv.Itoa(len(taskrunList.Items)), strings.Join(s, ", "))
```

Comment:
instead of calling ` strconv.Itoa` we could use the `%d` format operator (%s is forcing us to use a string: https://golang.org/pkg/fmt/#hdr-Printing)

Reply:
replaced it, thanks!